musicbrainz/csv/fetch.py

```diff
@@ -0,0 +1,59 @@
+"""
+fetch the latest test data dump files
+"""
+
+import os
+import time
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_latest_json_dump_url():
+    """
+    get the latest repo
+    """
+    url = "https://data.metabrainz.org/pub/musicbrainz/data/json-dumps/"
+    response = requests.get(url, timeout=50)
+    soup = BeautifulSoup(response.text, "html.parser")
+
+    # Extract all directories
+    dirs = [a.text for a in soup.find_all("a") if a.text.startswith("latest-is-")]
+
+    latest_dir = dirs[-1]
+    latest_dir = latest_dir[10:]
+    resp_url = url + latest_dir + "/"
+    return resp_url
```

Comment:
You could get this more simply and without all of the magic number string manipulation with:

`requests.get("https://data.metabrainz.org/pub/musicbrainz/data/json-dumps/LATEST").text.strip()`